server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -1431,21 +1436,29 @@ export class AgenticChatController implements ChatHandlers {
         } else {
             chatResultStream.setMessageIdToUpdateForTool(toolUse.name!, messageIdToUpdate)
         }
+        let currentPaths = []
+        if (toolUse.name === 'listDirectory' || toolUse.name === 'fileSearch') {
```

Comment:
should we do the if the other way so it checks for fsRead specifically? if we added more tools, we'll have to keep updating this list or it might fail?

Reply:
+1

Reply:
done